Signal/src/ViewControllers/ConversationView/ConversationViewController.m

```diff
@@ -483,6 +487,18 @@ - (void)createContents
     [self.inputToolbar autoPinWidthToSuperview];
     [self.inputToolbar autoPinEdge:ALEdgeTop toEdge:ALEdgeBottom ofView:self.collectionView];
     [self autoPinViewToBottomGuideOrKeyboard:self.inputToolbar];
+
+    self.loadMoreHeader = [UIButton buttonWithType:UIButtonTypeCustom];
```

Comment:
I think this increase the hit area of the "load more" header to be the entire header and not just the text.